sass/theme/_theme.scss

```diff
@@ -451,3 +451,11 @@ footer #wrapper-footer-innovation .img-link{
 			padding-right: 0.15em;
 	}
 }
+
+
+/*--------------------------------------------------------------
+Always set the image height in core/image to auto
+--------------------------------------------------------------*/
+.wp-block-image img{
+	height: auto!important;
```

Comment:
Any particular reason for the `! important` tag here? 

There are some edge cases in which the responsive image settings would want to be enforced with `auto:width` instead of an `auto:height;` The important declaration would make it harder to override this elsewhere if desired.